mobile/src/bundles/talent/components/cv-and-contacts-form/cv-and-contacts-form.tsx

```diff
@@ -2,16 +2,18 @@ import React from 'react';
 
 import {
     Button,
+    FilePicker,
     FormField,
     Input,
     ScrollView,
     Text,
     View,
 } from '~/bundles/common/components/components';
-import { IconName, TextCategory } from '~/bundles/common/enums/enums';
+import { TextCategory } from '~/bundles/common/enums/enums';
 import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/styles';
 import { AvatarPicker } from '~/bundles/talent/components/avatar-picker/avatar-picker';
+import { CvAndContactsFormValidationSchema } from '~/bundles/talent/validation-schemas/validation-schemas';
```

Comment:
Does the frontend use different validation rules?

Reply:
The validation was moved from the shared folder to the frontend folder. It is mostly the same, but there might be some differences for CV and image.